plugin/trino-influxdb/src/main/java/io/trino/plugin/influxdb/config/InfluxConfig.java

```diff
@@ -0,0 +1,200 @@
+    {
+        InfluxConfig config = new InfluxConfig();
+        config.setHost("localhost");
+        config.setPort(8086);
+        config.setProtocol(HTTP);
+        return config;
+    }
+
+    public String getHost()
+    {
+        return host;
+    }
+
+    @Config("influx.host")
+    public InfluxConfig setHost(@NotEmpty String host)
+    {
+        this.host = host;
+        return this;
+    }
+
+    public Integer getPort()
+    {
+        return port;
+    }
+
+    @Config("influx.port")
+    public InfluxConfig setPort(Integer port)
+    {
+        this.port = port;
+        return this;
+    }
+
+    public Protocol getProtocol()
+    {
+        return protocol;
+    }
+
+    @Config("influx.protocol")
+    public InfluxConfig setProtocol(Protocol protocol)
+    {
+        this.protocol = protocol;
+        return this;
+    }
+
+    public String getEndpoint()
+    {
+        return protocol.getProtocol() + "://" + HostAddress.fromParts(host, port);
+    }
+
+    public String getUsername()
+    {
+        return username;
+    }
+
+    @Config("influx.username")
+    public InfluxConfig setUsername(@NotEmpty String username)
+    {
+        this.username = username;
+        return this;
+    }
+
+    public String getPassword()
+    {
+        return password;
+    }
+
+    @ConfigSecuritySensitive
+    @Config("influx.password")
+    public InfluxConfig setPassword(String password)
+    {
+        this.password = password;
+        return this;
+    }
+
+    @MinDuration("1s")
+    @MaxDuration("1d")
+    public Duration getCacheTimeOut()
+    {
+        return cacheTimeOut;
+    }
+
+    @Config("influx.cacheTimeOut")
+    public InfluxConfig setCacheTimeOut(Duration cacheTimeOut)
+    {
+        this.cacheTimeOut = cacheTimeOut;
+        return this;
+    }
+
+    @MinDuration("3s")
+    @MaxDuration("3h")
+    public Duration getConnectTimeOut()
+    {
+        return Optional.ofNullable(connectTimeOut).orElse(new Duration(10, SECONDS));
+    }
+
+    @Config("influx.connectTimeOut")
+    public InfluxConfig setConnectTimeOut(Duration connectTimeOut)
+    {
+        this.connectTimeOut = connectTimeOut;
+        return this;
+    }
+
+    @MinDuration("3s")
+    @MaxDuration("3h")
+    public Duration getWriteTimeOut()
+    {
+        return Optional.ofNullable(writeTimeOut).orElse(new Duration(10, SECONDS));
+    }
+
+    @Config("influx.writeTimeOut")
+    public InfluxConfig setWriteTimeOut(Duration writeTimeOut)
+    {
+        this.writeTimeOut = writeTimeOut;
+        return this;
+    }
+
+    @MinDuration("3s")
+    @MaxDuration("3h")
+    public Duration getReadTimeOut()
+    {
+        return Optional.ofNullable(readTimeOut).orElse(new Duration(10, SECONDS));
+    }
+
+    @Config("influx.readTimeOut")
+    public InfluxConfig setReadTimeOut(Duration readTimeOut)
+    {
+        this.readTimeOut = readTimeOut;
+        return this;
+    }
+
+    public Boolean getHasTimeField()
```

Comment:
It only used in tpch test, because influx-client will return an additional field "time" in query result,  so i want to remove this field in metadata api.  Now i set it "true" when InfluxQueryRunner call 
> queryRunner.createCatalog("influxdb", "influxdb", properties);   
Is there any other way ?

Reply:
Why do you want to remove the field? How about making the field a hidden column?

Reply:
In connector test , expected result in some query , such as show columns from table "nation"、"orders" do not get "time" field. so i think i have to remove it to keep same schema with tpch tables.

> How about making the field a hidden column?

It okay , how to do this?

Reply:
Please refer to existing connectors. e.g. MongoDB